apps/laboratory/src/components/Bitcoin/BitcoinTests.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react'
+import { useAppKitAccount } from '@reown/appkit/react'
+import { StackDivider, Card, CardHeader, Heading, CardBody, Box, Stack } from '@chakra-ui/react'
+import { BitcoinSignMessageTest } from './BitcoinSignMessageTest'
+import { BitcoinSendTransferTest } from './BitcoinSendTransferTest'
+import { BitcoinSignPSBTTest } from './BitcoinSignPSBTTest'
+
+export function BitcoinTests() {
+  const [ready, setReady] = React.useState(false)
+
+  const { isConnected } = useAppKitAccount()
+
+  React.useEffect(() => {
+    setReady(true)
+  }, [])
```

Comment:
Is it better to create `useIsMounted` hook ?

Reply:
I've removed this logic since it was not really needed. I did copy it from EthersTests component.